apps/ensrainbow/src/ingest.ts

```diff
@@ -51,30 +57,33 @@ async function loadEnsNamesToLevelDB(): Promise<void> {
       break;
     }
 
-    if (isCopySection) {
-      const parts = line.trim().split("\t");
-      if (parts.length === 2) {
-        const [hashVal, name] = parts;
-        if (hashVal && name) {
-          try {
-            // Convert hex string to Buffer, stripping '0x' prefix
-            const hashBytes = Buffer.from(hashVal.slice(2), "hex");
-            batch.put(hashBytes, name);
-            batchSize++;
-
-            if (batchSize >= MAX_BATCH_SIZE) {
-              await batch.write();
-              batch = db.batch();
-              batchSize = 0;
-            }
-            bar.tick();
-          } catch (e) {
-            console.error(`Error processing hash: ${e} '${hashVal}'`);
-          }
-        }
+    if (!isCopySection) {
+      continue;
+    }
+
+    const parts = line.trim().split("\t");
+    if (parts.length !== 2) {
+      continue;
+    }
+
+    const [hashVal, name] = parts;
```

Comment:
```suggestion
    const [labelHash, label] = parts;
```

I assume the above is more accurate?